src/products/iOS_products/review_iOSstrings.md

```diff
@@ -1,70 +1,74 @@
 
 Note that these will not appear as new strings for localizers and they will not need to touch them.
 
 Changes to comments (`<note>`) are irrelevant in terms of string updates, so they’re also OK.
 
-### Potentially problematic changes (removals)
+### Potentially problematic changes
 
-Then you start scrolling down, checking if the new strings make sense: the first one you find is `NSCameraUsageDescription`.
+Then you start scrolling down, checking if the new strings make sense: the first one you find is `CreditCard.DisplayCard.ExpiresLabel.v115`.
 
-You mostly need to pay attention to the strings removed. Double-check each time that iOS team clearly realizes that they won’t be able to release updates to v5 (or any v5.x release) once strings are exported.
-
-For example: `Menu.NightModeAction.Title` is removed. If it’s used in v5, and the iOS team plans to release an update to v5 in the future, localizations will miss that string.
+Things to look out for:
+* Unclear strings and missing localization comments: the best way to identify them is to translate the strings, only having the string and comment as context (not the entire file, or the associated commit where the string landed). For example: is the word used both a noun and a verb in English? Is the ID clear enough to give context (e.g. `buttonLabel`)?
+* String changes without new IDs. IDs in XLIFF files are stored in the `trans-unit` attribute; for example, the ID of [this string](https://github.com/mozilla-l10n/firefoxios-l10n/pull/192/files#diff-6f29eb799be1b575316c0187d69a38fce3c63e27e5a22eb180b338ee177e9caeR3201) is `CreditCard.DisplayCard.ExpiresLabel.v115`.
+* Duplicated strings.
+* Other general [localization issues](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html).
 
 ### Quickly review other locales
 
-Once you’ve checked `templates`, you can pick at least one locale and see what changes, as you can see [for example here](https://github.com/mozilla-l10n/firefoxios-l10n/pull/19/commits/a1c041a012e6996bd8bdaec88f21f3c34fe383b0).
+Once you’ve checked `templates`, you can pick at least one locale and see what changes, as you can see [for example here](https://github.com/mozilla-l10n/firefoxios-l10n/pull/192/files#diff-220a1dc4ddc01bbae6a176ba4122aa644042181c78a638b052f45462b758ca6f).
+
+Checking another locale helps to easily spot if there are any file removals that might be deleting entire sections of existing translations. It can also help spot instances where existing translations are accidentally removed or reverted. See for example this [past PR](https://github.com/mozilla-l10n/firefoxios-l10n/pull/124), where engineers did not import the latest localizations before exporting again strings from their code repository side.
```

Comment:
The first part is OK, the second is potentially misleading. That dates back to a time when the developer had to manually run scripts on their device, these days are gone.